cabal.project

```diff
@@ -1,6 +1,6 @@
 packages: large-generics
         , large-records
-        , large-records-benchmarks
+--        , large-records-benchmarks
```

Comment:
hmm, that shouldn't been commented out

Reply:
Oh, oops :) I guess apart from making sure the benchmarks still compile, it would also be good to run then so that there are no obvious regressions. I can do that if you like.

Reply:
Please do